lib/api/tensor-impl.ts

```diff
@@ -26,7 +26,7 @@ export class Tensor implements TensorInterface {
         this.data = Float32Array.from(data as number[]);
       } else if (type === 'bool') {
         // convert boolean[] to Uint8Array
```

Comment:
Add a comment to use new instead of from method here. When safari supports Uint8Array.from, this can be changed for consistency with other lines.